pkgs/misc/tmux-plugins/default.nix

```diff
@@ -90,6 +90,27 @@ in rec {
     };
   };
 
+  catppuccin = mkTmuxPlugin {
+    pluginName = "catppuccin";
+    version = "e2561de";
```

Comment:
Please use the unstable format, see below how to use it

Reply:
Ah nice. Done, thanks!